controller/lib/src/adp.h

```diff
@@ -82,6 +82,13 @@ class adp
     ///
     /// Get the Controller Entity ID of the AVDECC Entity sending the command.
     ///
+    /// This function is used by descriptor commands to fetch the
+    /// controller entity id corresponding to a particular end station.
```

Comment:
Technically an end station can have multiple roles; any combination of multiple talkers, listeners, responders, controllers.  Each one needs a different EUI-64; typically you would use the middle two bytes to specify a unique number for each entity on the same end station.

--jeffk++


> On Jul 26, 2016, at 8:50 AM, asi-build <notifications@github.com> wrote:
> 
> In controller/lib/src/adp.h:
> 
> > @@ -82,6 +82,13 @@ class adp
> >      ///
> >      /// Get the Controller Entity ID of the AVDECC Entity sending the command.
> >      ///
> > +    /// This function is used by descriptor commands to fetch the
> > +    /// controller entity id corresponding to a particular end station.
> This text does not make sense. An end station does not have a controller entity id.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>